samples/samples.py

```diff
@@ -0,0 +1,48 @@
+import json
+from pprint import pprint
+
+from azure.identity import InteractiveBrowserCredential
```

Comment:
>azure.identity [](start = 5, length = 14)

any update needed to requirements.txt for this?

Reply:
That depends on how users use the samples. I am not sure what the right assumption is here, do users download the repo and run samples or do they copy & paste samples from the repo. @MIchaelMainer can provide guidance.

Reply:
since it's part of the repo i'd just add it so anyone who pulls the repo just gets it working without having to worry about it.